tests/test_watcher_comprehensive.py

```
@@ -0,0 +1,574 @@
+        """Test that check_interval is validated."""
+        mock_config.get.side_effect = lambda s, k, **kw: 0 if (s, k) == ("Watcher", "check_interval") else mock_config.config.get(s, {}).get(k)
+
+        GengoWatcher(mock_config, mock_state, logger)
+        # Should have been corrected to minimum of 5
+        mock_config.set.assert_called()
+
+    def test_csv_logging_setup_when_enabled(self, mock_config, mock_state, logger):
+        """Test CSV logging setup."""
+        mock_config.get.side_effect = lambda s, k, **kw: True if (s, k) == ("Logging", "log_all_entries_enabled") else mock_config.config.get(s, {}).get(k, kw.get("fallback"))
+
+        with patch("builtins.open", mock_open()) as mock_file:
```

Ruff (0.14.14)

Local variable `mock_file` is assigned to but never used

Remove assignment to unused variable `mock_file`

(F841)